Buildings/Controls/OBC/Utilities/PIDWithAutotuning/FirstOrderAMIGO.mo

```diff
@@ -20,16 +20,16 @@ block FirstOrderAMIGO
     "Typical range of control error, used for scaling the control error";
   parameter Real yHig = 1
     "Higher value for the relay output";
-  parameter Real yLow = 0.1
+  parameter Real yLow
     "Lower value for the relay output";
-  parameter Real deaBan(min=1E-6) = 0.1
+  parameter Real deaBan(min=1E-6)
     "Deadband for holding the output value";
-  parameter Real yRef = 0.8
+  parameter Real yRef
     "Reference output for the tuning process";
```

Comment:
`Reference output for the tuning process. It cannot be greater or less than the higher or lower value of the relay output`